tests/runfiles/common.run

```diff
@@ -284,10 +284,11 @@ tests = ['cache_001_pos', 'cache_002_neg', 'canmount_001_pos',
     'zfs_set_feature_activation']
 tags = ['functional', 'cli_root', 'zfs_set']
 
+# Disabled - zfs_share_concurrent_shares
```

Comment:
Is there a reason not to add it as expected to fail, instead of total disable?

Reply:
Good point, fixed.  It turns out we'd already added this one to the list for FreeBSD.